cvat-ui/src/components/annotation-page/attribute-annotation-workspace/attribute-annotation-sidebar/attribute-editor.tsx

```diff
@@ -24,6 +28,71 @@ interface InputElementParameters {
     onChange(value: string): void;
 }
 
+const componentShortcuts = {
+    SET_0_VALUE: {
+        name: 'Set 0th value to the current attribute',
```

Comment:
What is 0th value?

Order is incorrect in this list.

0 -> first value
1 -> second value, etc.